src/tests/functional/shared_test_classes/src/subgraph/quantized_convolution_backprop_data.cpp

```diff
@@ -32,50 +38,60 @@ std::string QuantConvBackpropDataLayerTest::getTestCaseName(const testing::TestP
     result << "AP=" << padType << "_";
     result << "Levels=" << quantLevels << "_";
     result << "QG=" << quantGranularity << "_";
-    result << "netPRC=" << netPrecision.name() << "_";
+    result << "netPRC=" << element_type << "_";
     result << "targetDevice=" << targetDevice;
     return result.str();
 }
 
 void QuantConvBackpropDataLayerTest::SetUp() {
-    threshold = 0.5f;
+    // threshold = 0.5f;
 
     quantConvBackpropDataSpecificParams groupConvBackpropDataParams;
     std::vector<size_t> inputShape;
-    auto netPrecision = InferenceEngine::Precision::UNSPECIFIED;
-    std::tie(groupConvBackpropDataParams, netPrecision, inputShape, targetDevice) = this->GetParam();
-    ngraph::op::PadType padType;
+    ov::element::Type element_type;
+    std::tie(groupConvBackpropDataParams, element_type, inputShape, targetDevice) = this->GetParam();
+    ov::op::PadType padType;
     InferenceEngine::SizeVector kernel, stride, dilation;
     std::vector<ptrdiff_t> padBegin, padEnd;
     size_t convOutChannels;
     size_t quantLevels;
     QuantizationGranularity quantGranularity;
-    std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType, quantLevels, quantGranularity) = groupConvBackpropDataParams;
-    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-    ov::ParameterVector params{std::make_shared<ov::op::v0::Parameter>(ngPrc, ov::Shape(inputShape))};
-    auto paramOuts = ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
+    std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType, quantLevels, quantGranularity) =
+        groupConvBackpropDataParams;
+    ov::ParameterVector params{std::make_shared<ov::op::v0::Parameter>(element_type, ov::Shape(inputShape))};
+    auto paramOuts =
+        ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ov::op::v0::Parameter>(params));
```

Comment:
These ` ngraph::helpers::` and  ` ngraph::builder::` functions will be replaced by some 'ov' version?